### PR TITLE
Fieldset heading levels

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
### Context

Use correct heading levels in fieldsets/correctly use label for heading on missing work history field

### Link to Trello card

[460 - Tagging headings card h1s h2s h3s](https://trello.com/c/HBER52SE/)

### CODE REVIEW

File: config/locales/application_form.yml [hunk 1]
```diff
@@ -436,11 +440,6 @@ en:
             end_date:
               invalid: Enter an end date in the correct format
               in_the_future: Enter an end date that is not in the future
-        candidate_interface/work_breaks_form:
-          attributes:
-            work_history_breaks:
-              blank: Tell us about any breaks in your work history
-              too_long: Explanation of any breaks in your work history must be %{count} characters or fewer
```
Comment (paulrobertlloyd) [inline config/locales/application_form.yml]: `too_long` is not actually called. [Created a related card](https://trello.com/c/nCljMNvt/562-error-for-word-count-should-include-maximum-word-count)

Comment (chubberlisk) [reply config/locales/application_form.yml]: Wondering if we need a character limit _and_ a word limit?

File: app/views/candidate_interface/gcse/type/edit.html.erb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
       <%= f.govuk_error_summary %>
 
-      <%= f.govuk_radio_buttons_fieldset :qualification_type, legend: { size: 'm', text: t('application_form.gcse.qualification_type.label') } do %>
+      <%= f.govuk_radio_buttons_fieldset :qualification_type, legend: { size: 'm', text: t('application_form.gcse.qualification_type.label'), tag: 'span' } do %>
```
Comment (fofr) [inline app/views/candidate_interface/gcse/type/edit.html.erb]: Any reason not to use h2s or h3s?

Comment (paulrobertlloyd) [reply app/views/candidate_interface/gcse/type/edit.html.erb]: Surely `<legend>` provides the required structure and semantics?

**Pull request merged**